Ahmedq02/task-app
# Add docker

# Add Docker



## Description

Dockerize the app.

- **Related Issue:** Closes #9 
- **Summary of Changes:**
  - Add Dockerfile
  - Add docker-compose.yaml file
  - Add entry scripts

## How to Test



1. Check out this branch: `git checkout <branch-name>`
2. Dockerize app `docker compose up --build`
3. Test the app runs.
